tests/protocol/test_wire.py

```diff
@@ -515,6 +515,19 @@ def test_get_ext_conf_with_extensions_should_retrieve_ext_handlers_and_vmagent_m
             self.assertFalse(extensions_goal_state.on_hold,
                               "Extensions On Hold is expected to be False")
 
+    def test_it_should_parse_requested_version_properly(self):
```

Comment:
this test, or part of it, should be merged into  ExtensionsGoalStateFromVmSettingsTestCasetest_create_from_vm_settings_should_parse_vm_settings

Reply:
This one is for testing the Fabric goal state. I'll move it to the `ExtensionsGoalStateFromExtensionsConfigTestCase` class

Reply:
Done